src/components/shmoke.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+<div id='s' class="fl ma2 tc b--solid bg-light-green shadow-5 db pa3"> 
+        <h2>🌲 SHMOKE up FELLA 🌲 </h2>
+        <img class="snoop" @click="shmoke_weed"  :src = getimageUrl() />
+</div>     
+
+</template>
+
+
+<script>
+import { METHODS } from 'http';
+import eventBus from '../eventBus';
+export default {
+    data(){
+        return {
+            shmoking: [
+                {
+            shmoked:true, 
+            name:"shmokie",
```

Comment:
Why `"shmokie"` why not `"shmoke"`?

Reply:
CAUSE ITS SHMOKIE IN HERE (but i didnt want everything to be named shmoke to confuse myself )